stan/math/prim/mat/fun/to_matrix.hpp

```diff
@@ -70,6 +70,31 @@ namespace stan {
 
 
     /**
+     * Returns a matrix representation of the Eigen vector of standard vectors
+     * with the same dimensions and indexing order.
+     *
+     * @tparam T type of the scalar
+     * @param vec Eigen vector of vectors of scalar values
+     * @return the matrix representation of the input
+     */
+    template <typename T>
+    inline Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>
+    to_matrix(const std::vector< Eigen::Matrix<T, 1, Eigen::Dynamic> >& vec) {
+      size_t R = vec.size();
+      if (R != 0) {
+        size_t C = vec[0].size();
+        Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> result(R, C);
+        T* datap = result.data();
```

Comment:
This is much simpler.

```
for (int i = 0; i < result.rows(); ++i)
  result.row(i) = vec[i];
```

But I don't think that arg should be called `vec` --- just use a dummy variable like `x` instead of an abbreviated typename.